lavaplay/player.py

```diff
@@ -33,8 +33,8 @@ def add_to_queue(self, tracks: t.List[Track], requester: t.Optional[int] = None)
         """
         Add tracks to queue. use to load a playlist result.
 
-        >>> playlist = lavaplayer.search_youtube("playlist url")
-        >>> lavaplayer.add_to_queue(playlist.tracks)
+        >>> playlist = lavaplay.search_youtube("playlist url")
+        >>> lavaplay.add_to_queue(playlist.tracks)
```

Comment:
Enhance the example

Reply:
Ah, I hadn't noticed the change. What do you mean, I didn't understand?